core/inc/com/centreon/broker/broker_impl.hh

```diff
@@ -37,7 +37,9 @@ class broker_impl final : public Broker::Service {
   grpc::Status GetSqlConnectionStats(grpc::ServerContext* context,
                            const ::google::protobuf::Empty* request,
                            BrokerStats* response) override;
```

Comment:
I would prefer a SqlConnnectionStats pointer as output.
And In input, we should be able to give an index as int.